packages/ui-stencil/src/components/orama-search-box/orama-search-box.scss

```diff
@@ -23,6 +23,12 @@
   }
 }
 
+.embed {
+  display: block;
+  background-color: var(--background-color-primary, background-color(primary));
+  border-radius: var(--radius-m,calc(12rem/var(--orama-base-font-size, 16)))
```

Comment:
you can replace
`calc(12rem/var(--orama-base-font-size, 16))`
with
`$radius-m`
(it already covers calculations)